htdocs/modules/debugbar/assets/css/font-awesome.css

```diff
@@ -13,7 +13,7 @@
 }
 .fa {
   display: inline-block;
-  font-family: FontAwesome;
+  font-family: FontAwesome, sans-serif;
```

Comment:
We supply the font, and if for some reason it doesn't make it, debugbar will look broken. The fallback won't help at all.